plugins/UsersManager/templates/index.twig

```diff
@@ -178,6 +178,7 @@
                 {% if showLastSeen is defined and showLastSeen %}
                 <th>{{ 'UsersManager_LastSeen'|translate }}</th>
                 {% endif %}
+                <th>token_auth</th>
```

Comment:
We could have a separate content block for it but it could be a long list when there are > 30k users. We would need to have a user select field or a user login auto complete field similar as we have for "give view access" feature under "Manage access". It is only visible to users with admin access but not to users with super user access. We could later reuse the same logic from "give view access" .

The whole users manager needs a big re-design eventually. For now it would be good to just move that big button out and maybe replace it with a tiny icon in the existing column. In another PR we could add a new content block where users can type in the login name or email address and then re-generate their token